### DISCUSSION TITLE
Number of validation windows

### DISCUSSION THREAD

**Discussion by michelidiego** [autogluon/autogluon#3391] [Q&A] [answered]
Hi, I was wondering if there's a "best practice" recommendation for the number of validation windows when forecasting using time series. I think the default value is 3 but I'm not sure on how this affects the fitting process. 

thanks

**Comment by shchur** [comment, marked as answer]
Hi, the default value in v0.8 is a single validation window to handle cases where the training dataset is very short (before we implement more sophisticated logic to handle this case). 

In general, I would recommend ensuring that the shortest time series in `train_data` has length of at least `(2 + num_val_windows) * prediction_length + 1` for whatever value of `num_val_windows` you choose.

Note that the training time increases by approximately by the factor of `num_val_windows` as you increase this parameter.

**Reply by michelidiego** [reply]
Thank you very much @shchur . You've been really helpful
